### DISCUSSION TITLE
Why does the `Select` element only allow for a `IEnumberable` to be binded?

### DISCUSSION THREAD

**Discussion by lettuceDestroyer** [ant-design-blazor/ant-design-blazor#3840] [General]
When trying to bind a `List<T>` to the `AntDesign.Select` element I get the error mesages:

```
Error (active) CS1662: Cannot convert lambda expression to intended delegate type because some of the return types in the block are not implicitly convertible to the delegate return type
```

```
Error (active) CS1503 cannot convert from 'Microsoft.AspNetCore.Components.EventCallback<System.Collections.Generic.List<SomeDTO>>' to 'Microsoft.AspNetCore.Components.EventCallback'
```

Why is that? Isn't it possible to `@bind-Value` a list? Since I get the data from a API and use DTOs it is not possible to change the data type of that property. Now I have to create a method which is binded to the forms `OnFinishFailed` callback. This method then sets the properties of my DTO, validates the form again and submits it. This makes the code quite ugly and hard to understand.

**Comment by ElderJames** [comment]
This is because the ValueChanged need to assign the IEnumerable<T> to value.
